includes/class-wc-payments-order-service.php

```diff
@@ -216,7 +216,7 @@ public function mark_payment_capture_failed( $order, $intent_id, $intent_status,
 	}
 
 	/**
-	 * Updates an order to cancelled status, while adding a note with a link to the transaction.
+	 * Update an order to failed status, and add note with a link to the transaction. Context - when a Payment Intent expires. Changing the status to failed will enable the buyer to re-attempt payment.
```

Comment:
```suggestion
	 * When a payment intent expires update an order to failed status, and add note with a link to the transaction.
	 *
	 * Context: changing the status to failed will enable the buyer to re-attempt payment.
```

Docblock is expected to have a brief summary on the first line follow-ed up by a detailed overview if needed.

Reply:
Thanks Valery!. I have changed it here - b4a4c65